jfrog_platform/manifest.json

```
@@ -24,7 +24,7 @@
   "integration_id": "jfrog-platform",
   "assets": {
     "dashboards": {
-      "Jfrog Artifactory Dashboard": "assets/dashboards/jfrog_artifactory_dashboard.json"
+      "JFrog Artifactory Dashboard": "assets/dashboards/jfrog_artifactory_dashboard.json"
```

```suggestion
      "Jfrog Artifactory Dashboard": "assets/dashboards/jfrog_artifactory_dashboard.json"
```
This key cannot be changed (it's used for the backend) and has no impact on the dashboard title.